src/main/java/user/createCorporatePage/PageGenerateController.java

```diff
@@ -0,0 +1,54 @@
+package user.createCorporatePage;
+
+
+import entity.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class PageGenerateController {
+    final IPageGenerateInput interactor;
+
+    @Autowired
+    public PageGenerateController(IPageGenerateInput interactor) {
+        this.interactor = interactor;
+    }
+
+    @PostMapping
```

Comment:
You need to give the mapping a route, for example, it could be POST to `/pages`

Reply:
addressed